common/test/acceptance/tests/lms/test_lms_user_preview.py

```diff
@@ -266,28 +266,26 @@ def populate_course_fixture(self, course_fixture):
             </problem>
         """)
 
+        self.alpha_text = "VISIBLE TO ALPHA"
+        self.beta_text = "VISIBLE TO BETA"
+        self.everyone_text = "VISIBLE TO EVERYONE"
+
         course_fixture.add_children(
             XBlockFixtureDesc('chapter', 'Test Section').add_children(
                 XBlockFixtureDesc('sequential', 'Test Subsection').add_children(
-                    XBlockFixtureDesc(
-                        'problem', 'Visible to alpha', data=problem_data, metadata={"group_access": {0: [0]}}
-                    ),
-                    XBlockFixtureDesc(
-                        'problem', 'Visible to beta', data=problem_data, metadata={"group_access": {0: [1]}}
-                    ),
-                    XBlockFixtureDesc('problem', 'Visible to everyone', data=problem_data)
+                    XBlockFixtureDesc('vertical', 'Test Unit').add_children(
```

Comment:
Note to reviewers: when I wrote this test, I followed the pattern of other tests in the file that put problems directly under subsections. I have changed it now to represent the structure that would be created by Studio (problems in a unit).

Because of that, I can't use course_nav.sequence_items, which only returns the display name of the first component on that page (not the display names of all components). If I put each problem into its own unit it also doesn't work because all students will see each unit, even if it has no content for that student (and the display name is the non-visible problem...).